pom.xml

```diff
@@ -70,7 +70,7 @@
         <git-plugin.version>3.1.0</git-plugin.version>
         <workflow-support-plugin.version>3.2</workflow-support-plugin.version>
         <scm-api-plugin.version>2.2.6</scm-api-plugin.version>
-        <groovy-cps.version>1.26</groovy-cps.version>
+        <groovy-cps.version>1.27-20190424.002713-11</groovy-cps.version> <!-- TODO https://github.com/cloudbees/groovy-cps/pull/85 -->
```

Comment:
Would love to set up Incrementals upstream, but that would probably require getting it moved into @jenkinsci, and would require some help from infrastructure people.